test/test_ParticleDistributions_correctness.jl

```diff
@@ -75,10 +75,10 @@ dist = GammaPrimitiveParticleDistribution(1.0, 1.0, 2.0)
 
 # Update params from moments
 dist_dict = Dict(:dist => dist)
-dist = update_params_from_moments(dist_dict, [1.1, 2.0, 4.1])
-@test moment(dist, 0.0) ≈ 1.1 rtol=rtol
+dist = update_params_from_moments(dist_dict, [1.1, 2.0, 4.1], Dict("θ" => (1e-5, 1e5), "k" => (eps(Float64), 5.0)))
+@test moment(dist, 0.0) ≈ 1.726 rtol=rtol
 @test moment(dist, 1.0) ≈ 2.0 rtol=rtol
-@test moment(dist, 2.0) ≈ 4.1 rtol=rtol
+@test moment(dist, 2.0) ≈ 2.782 rtol=rtol
```

Comment:
Need to add @test and @test_throws for newly added particle distribution functionality